doc/dev/design/ui_goals.rst

```diff
@@ -0,0 +1,54 @@
+Ceph Dashboard Design Goals
+===========================
+
+.. note:: this document is intended to provide a focal point for discussing the overall design principles for mgr/dashboard
+
+Introduction
+############
+
+Most distributed storage architectures are inherently complex, and can present a management challenge to Operations teams who are
+typically stretched across multiple product and platform disciplines. In general terms, the complexity of any solution can have a 
+direct bearing on the operational costs incurred to manage it. The answer is simple...make it simple :)
+
+This document is intended to highlight Ceph Dashboard design goals which may help to  
+
+* reduce complexity
+* increase productivity
+* improve time-to-value
+
+ 
+Understanding the Persona of the Target User
+############################################
+
+Ceph has historically been administered from the CLI. The CLI has always and will always offer the richest, most flexible way to 
+install and manage a Ceph cluster. Administrators who require and demand this level of control are unlikely to adopt a UI for anything
+more than a technical curiousity.
+
+The relevance of the UI is therefore more critical for a new SysAdmin, where it can help technology adoption and reduce the operational friction
+that is normally experienced when implementing a new solution.
+
+Understanding the target user persona is therefore a fundamental first step in design. Attempting to design a UI that meets the requirements of a 'seasoned'
+Ceph Administrator or Developer, and a relatively new SysAdmin is unlikely to satisfy either user group.
+
+Design Principles
+#################
+
+Key Principles
+--------------
+
+
+1. **Clarity and consistency**. The UI should ensure the data shown is unambiguous and consistent across different views
+2. **Data timeliness**. Data displayed in the UI must be timely. State information must always be reasonably recent for it to be relevant and acted upon with confidence.
```

Comment:
With the exception of RGW buckets and users (which are not periodically refreshed) everything else in dashboard is refreshed every around 5 secs.

Reply:
as long as the data it relies in is available though right? For example, the orch has a 10min event loop, when that expires it kicks off the refresh of daemon state, disks etc - so if the dashboard is dependent on this cache, it's only as good as the data it's given.

Reply:
Yes, you're right. A 10 min loop is clearly going to change the responsiveness feeling in the dashboard. The above mentioned parts of the dashboard (RGW buckets and users) need to be manually refreshed by the users (having them periodically polling might result in performance killing RGW side).